docs/examples/bicepconfig.json

```diff
@@ -2,6 +2,9 @@
   "analyzers": {
     "core": {
       "rules": {
+        "adminusername-should-not-be-literal": {
+          "level": "warning"

```

Comment:
Should we default new linter rules to on? It has the potential to break anyone who is treating warnings as errors and gets auto-upgraded (e.g. when using az cli / az powershell in a pipeline). Feels like maybe a DSL discussion topic?